test/heex_formatter_test.exs

```diff
@@ -202,12 +202,65 @@ defmodule HeexFormatterTest do
       """
       <section>
         <%= live_redirect to: "url", id: "link", role: "button" do %>
-        <div>
-          content
-        </div>
+          <div>
+            content
+          </div>
         <% end %>
       </section>
       """
     )
   end
+
+  test "proper format EEx tags" do
```

Comment:
@paulo-silva I added this test case and changed the previous one to guide us on how this should be formatted. Please, take a look and see if that makes sense to you as well.

Reply:
BTW, we might want to remove the previous test and keep this instead 🤔

Reply:
yeah, makes sense, I got to render the remaining tags like `middle_expr` that is like `<% else %>` and so on, I'm having issue now with inline expression, where we have an HTML tag and inside of it an eex expr, so, for example, the `p` that renders the product.name, the eex is rendered below the close tag, thoughts of what I could be doing wrong? 😅 

```html
<section>
  <ul>
    <%= for product <- @products do %>
    <li>
      <img src={optimized_image_url(product.image_url)} />
      <div class=\"px-5 mb-12\">
        <p>
        </p>
        <%= product.name %>
        <a href={product.external_url}>
          Product link
        </a>
        <div class=\"w-full absolute bottom-0\">
          <%= if is_nil(product.reserved_for) do %>

          <%= live_patch \"Book\", to: Routes.store_index_path(@socket, :edit, product.id) %>
          <button>
            Reserved
          </button>
          <% end %>
        </div>
      </li>
      <% end %>
    </ul>
</section>
```

Reply:
all the other stuffs are working except for the format for now 👍

Reply:
We have to do something in this direction: https://github.com/feliperenan/heex_formatter/pull/2/commits/42e00e18e7db3cc1325679e3c2a2e25bc8a9fe27

There are two issues that I still need to handle:

1) `if else` blocks are not being formatted properly.

2) There is something wrong with the order when combining eex nodes with HTML nodes

Given this input here:
```
<p>
<%=
@user.name
%></p>
```

We are getting this result:

```
<p>
</p>
<%= @user.name %>
```

instead of 

```
<p>
  <%= @user.name %>
</p>
```